File: desafios_de_codigo/javaScript/desafio.js
//Lê os valores de entrada
const ValorSalario = parseFloat(gets());
const ValorBeneficios = parseFloat(gets());

//Calcula o imposto através da função 'calcularImposto'
const valorImposto = calcularImposto(ValorSalario);
//Calcula e imprime a Saída (com 2 casas decimais)
const saida = ValorSalario - valorImposto + ValorBeneficios;
print(saida.toFixed(2));

//Função útil para o calculo de imposto(baseado nas aliquotas)
function calcularImposto(salario){
    let aliquota;
    if (salario >= 0 && salario <= 1000){
        aliquota = 0.05;
    }else if (salario >= 1100.01 && salario <= 2500.00){
        aliquota = 0.10 * salario;
    }else{
        aliquota = 0.15* salario;
    }
    return aliquota * salario
}

File: desafios_de_codigo/python/desafio.py
# Função util para o calculo do imposto (baseados nas aliquotas)
def calcular_imposto(salario):
    aliquota = 0.00
    if ( salario >= 0 and salario <= 1100):
        aliquota = 0.05
    elif (salario >= 1100.01 and salario <= 2500.00):
            aliquota = 0.10 * salario
    else:
            aliquota = 0.15 * salario
    return aliquota

# Le os valores de Entrada
valor_salario = float(input())
valor_beneficios = float(input())

# Calcula p imposto através da função "calcular_imposto"
valor_imposto = calcular_imposto(valor_salario)
# Calcula e imprime a Saída ( com 2 casas decimais):
saida = valor_salario - valor_imposto + valor_beneficios
print(f'{saida:.2f}')